ui/media/js/main.js

```diff
@@ -1546,15 +1617,21 @@ heightField.addEventListener("change", onDimensionChange)
 
 function renameMakeImageButton() {
     let totalImages =
-        Math.max(parseInt(numOutputsTotalField.value), parseInt(numOutputsParallelField.value)) * getPrompts().length
+        Math.max(parseInt(numOutputsTotalField.value), parseInt(numOutputsParallelField.value)) * getPromptsNumber()
     let imageLabel = "Image"
     if (totalImages > 1) {
         imageLabel = totalImages + " Images"
     }
     if (SD.activeTasks.size == 0) {
-        makeImageBtn.innerText = "Make " + imageLabel
+        if (totalImages >= 10000)
+            makeImageBtn.innerText = "Make tens of thousands of images"
```

Comment:
How about "Make 10000+ images", to prevent the button from becoming too long (especially on mobile, which many users use)? Thanks